src/ui/src/components/core/base/BaseInputRange.vue

```diff
@@ -64,6 +64,10 @@ const thumbRadius = 9;
 
 const thumb = ref<HTMLElement>();
 const slider = ref<HTMLElement>();
+const displayValue = computed(() => {
+	const precision = Math.ceil(-Math.log10(props.step));
+	return model.value.toFixed(precision);
+});
```

Comment:
I would use two computed values, because it avoids to re-compute `precision` when value change
```suggestion
const precision = computed(() => Math.ceil(-Math.log10(props.step)));
const displayValue = computed(() => model.value.toFixed(precision.value));
```